apps/devhub/tests/test_views.py

```diff
@@ -2483,6 +2483,45 @@ def test_unlisted_addon_pass_validation_with_flag(self, mock_sign_file):
         assert file_.status == amo.STATUS_LITE
         assert mock_sign_file.called
 
+    @mock.patch('devhub.views.sign_file')
+    def test_beta_addon_pass_validation_with_flag(self, mock_sign_file):
+        """Beta files that pass validation are automatically signed/reviewed if
+        the flag is enabled."""
+        self.create_flag('automatic-validation')
+        version = self.addon.latest_version
+        version.all_files[0].update(status=amo.STATUS_BETA)
```

Comment:
This causes the test to fail on line 2497 because the addon having no reviewed file (only beta files) will have its status reset to "unreviewed".

You're testing that the file you're posting is set as beta and signed, you don't care about the existing files, so you should simply remove this line.